src/resource_pool_texture.cpp

```diff
@@ -22,6 +22,10 @@ namespace wr
 			{
 				texture = LoadPNG(path, srgb, generate_mips);
 			}
+			else if (ext_int.find("jpeg") != std::string_view::npos)
```

Comment:
find jpeg. loadpng. Something should be changed here :)

Reply:
Fixed in [https://github.com/TeamWisp/Procedural-Ray-Tracing/pull/194/commits/3ff74f640cdf84cb53edc0bc8196ae274350507e](this commit)